content/product-engineering/engineering/enablement/frontend-platform/index.md

```diff
@@ -63,20 +63,20 @@ We inherit Sourcegraph's [engineering principles and practices](../../principles
 
 ### Planning and prioritization
 
-We plan and track our day-to-day work on our [Kanban board](https://github.com/orgs/sourcegraph/projects/144). Our current process (last updated 2021-06-29) is as follows:
-
-- Incoming tickets (e.g., from other teams) are given the `team/frontend-platform` label. As part of our [triage process](#triage), these tickets are added to our board by the designated teammate.
-- Work is scheduled by adding a card to either the _Backlog (product)_ or the _Backlog (eng)_ column.
-  - _Backlog (product)_ is for work that directly contributes to product goals and is refreshed and prioritized by the PM/EM.
-  - _Backlog (eng)_ is refreshed and prioritized by the engineers. This is the place for refactorings, developer experience improvements, etc.
-- Work should not be moved into either column until it is ready for development. Tickets that require further input before an engineer can start working on them are put in the _Needs input_ column.
-- Tickets in the _Needs input_ column should be tagged with an appropriate label that describes what input is required:
-  - `needs-design`: if it needs input from the design team
-  - `needs-discussion`: if the issue should be discussed at the next Frontend Platform sync
-  - `needs-more-info`: if it's not clear yet what needs to be done
-  - `needs-prioritization`: if the issue is part of an active prioritization conversation, to be decided on soon
-  - `awaiting-reply`: if we're waiting on an answer to a specific question on the ticket
-- When starting work, engineers pull cards from one of the _Backlog_ columns and move it to the _In Progress_ column. The other columns are self-explanatory 🙂
+We plan and track our day-to-day work with [Github projects](https://github.com/orgs/sourcegraph/projects/222/views/1). Our current process (last updated 2021-12-9) is as follows:
+
+- Incoming tickets (e.g., from other teams) must have the `team/frontend-platform` label applied. As part of our [triage process](#triage), these tickets are investigated and prioritized by the designated teammate.
+- Work is scheduled by assigning issues a status of _Backlog_ and a target of _This iteration_, _Next iteration_, _This quarter_ or _Not planned_.
+- The [Current work](https://github.com/orgs/sourcegraph/projects/222/views/1) view reflects work that is actively in progress.
+  - By EOD Monday (in advance of our Tuesday sync), devs should update issues and add assign a status of _On deck_ to the work they plan to pick up.
+  - Once work on an issue has begun, it should have a status of _In progress_.
+  - Work that requires input should have a status of _Needs input_ and the appropriate label and asignees. For example, an issue that needs design input should be assigned to [Alicja Suska](../../../../team/index.md#alicja-suska) and given the `needs-design` label. Other relevant labels:
```

Comment:
I think what we discussed here is that stuff that is waiting on design _shouldn't_ be in the _Needs input_ status, because _Needs input_ means "not yet triaged".

I'd say anything that needs input could be in _Backlog_, _On deck_, or _Waiting_ (as appropriate).